3.2/itop_design.xsd

```diff
@@ -2624,37 +2639,78 @@
             <xs:enumeration value="UR_ACTION_BULK_DELETE" />
         </xs:restriction>
     </xs:simpleType>
+    <xs:complexType name="alteredMenuEnableActionEnumeration">
+        <xs:simpleContent>
+            <xs:extension base="MenuEnableActionEnumeration">
+                <xs:attributeGroup ref="alteredNode" />
+            </xs:extension>
+        </xs:simpleContent>
+    </xs:complexType>
 
     <xs:simpleType name="MenuEnablePermissionEnumeration">
         <xs:restriction base="xs:string">
             <xs:enumeration value="UR_ALLOWED_NO" />
             <xs:enumeration value="UR_ALLOWED_YES" />
         </xs:restriction>
     </xs:simpleType>
+    <xs:complexType name="alteredMenuEnablePermissionEnumeration">
+        <xs:simpleContent>
+            <xs:extension base="MenuEnablePermissionEnumeration">
+                <xs:attributeGroup ref="alteredNode" />
+            </xs:extension>
+        </xs:simpleContent>
+    </xs:complexType>
```

Comment:
I would have preferred these to be combined into 1 type, but couldn't find it. Do you have an idea?